stories/bar/bar.stories.js

```diff
@@ -78,7 +109,8 @@ export const desktop = () => `
 
 desktop.parameters = {
     docs: {
-        iframeHeight: 200
+        iframeHeight: 200,
+        storyDescription: 'The default bar contains a back button, page title, segmented button and product switch button. It can be displayed by using the <code>fd-bar</code> class, and is responsive to desktop screen sizes.'
```

Comment:
Is the button on the far right called a "product switch"? I found the term here: https://experience.sap.com/fiori-design-web/tool-header/#with-side-navigation-menu-icon-title-and-caret

Look for the component labeled as **J**.

Reply:
Hey Jessie, the button on the far right is not called Product Switch. It is part of the Product Switch and it's role is to toggle the popover body containing the products or LoBs that form the Product Switch. You can take a look at it [here](https://fundamental-styles.netlify.app/?path=/docs/components-product-switch--product-switch-in-shellbar)

Reply:
Ah, okay. What would you call this button then? a menu icon?

Reply:
No idea honestly :D For me it's always been Product Switch button. @droshev any suggestions?